x/deployment/types/msgs.go

```diff
@@ -37,7 +37,12 @@ func (msg MsgCreateDeployment) GetSignBytes() []byte {
 
 // GetSigners defines whose signature is required
 func (msg MsgCreateDeployment) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{msg.ID.Owner}
+	owner, err := sdk.AccAddressFromBech32(msg.ID.Owner)
```

Comment:
Can you validate all addresses in `ValidateBasic()`?

Reply:
@boz @akhilkumarpilli, as we doing `msg.ID.Validate()` (which checks errors from sdk.AccAddressFromBech32) in all `ValidateBasic` can remove panic then